docs/index.rst

```diff
@@ -229,4 +237,4 @@ Cloud Storage
   client = storage.Client()
   bucket = client.get_bucket('<your-bucket-name>')
   blob = bucket.blob('my-test-file.txt')
-  blob.upload_from_string('this is test content!')
+  blob.upload_from_string('this is test content!')
```

Comment:
I checked out the branch. Confirmed it was trailing whitespace (or absence of it).